src/cascadia/TerminalApp/defaults.json

```diff
@@ -203,55 +203,55 @@
-        { "command": { "action": "switchToTab", "index": 2 }, "keys": ["ctrl+alt+3"] },
-        { "command": { "action": "switchToTab", "index": 3 }, "keys": ["ctrl+alt+4"] },
-        { "command": { "action": "switchToTab", "index": 4 }, "keys": ["ctrl+alt+5"] },
-        { "command": { "action": "switchToTab", "index": 5 }, "keys": ["ctrl+alt+6"] },
-        { "command": { "action": "switchToTab", "index": 6 }, "keys": ["ctrl+alt+7"] },
-        { "command": { "action": "switchToTab", "index": 7 }, "keys": ["ctrl+alt+8"] },
-        { "command": { "action": "switchToTab", "index": 8 }, "keys": ["ctrl+alt+9"] },
-        { "command": "find", "keys": [ "ctrl+shift+f" ] },
-        { "command": "toggleFullscreen", "keys": [ "alt+enter" ] },
-        { "command": "toggleFullscreen", "keys": [ "f11" ] }
+        { "command": "closePane", "keys": "ctrl+shift+w" },
+        { "command": "closeWindow", "keys": "alt+f4" },
+        { "command": "copy", "keys": "ctrl+shift+c" },
+        { "command": "copy", "keys": "ctrl+insert" },
+        { "command": "decreaseFontSize", "keys": "ctrl+-" },
+        { "command": "duplicateTab", "keys": "ctrl+shift+d" },
+        { "command": "increaseFontSize", "keys": "ctrl+=" },
+        { "command": { "action": "moveFocus", "direction": "down" }, "keys": "alt+down" },
+        { "command": { "action": "moveFocus", "direction": "left" }, "keys": "alt+left" },
+        { "command": { "action": "moveFocus", "direction": "right" }, "keys": "alt+right" },
+        { "command": { "action": "moveFocus", "direction": "up" }, "keys": "alt+up" },
+        { "command": "newTab", "keys": "ctrl+shift+t" },
+        { "command": { "action": "newTab", "index": 0 }, "keys": "ctrl+shift+1" },
+        { "command": { "action": "newTab", "index": 1 }, "keys": "ctrl+shift+2" },
+        { "command": { "action": "newTab", "index": 2 }, "keys": "ctrl+shift+3" },
+        { "command": { "action": "newTab", "index": 3 }, "keys": "ctrl+shift+4" },
+        { "command": { "action": "newTab", "index": 4 }, "keys": "ctrl+shift+5" },
+        { "command": { "action": "newTab", "index": 5 }, "keys": "ctrl+shift+6" },
+        { "command": { "action": "newTab", "index": 6 }, "keys": "ctrl+shift+7" },
+        { "command": { "action": "newTab", "index": 7 }, "keys": "ctrl+shift+8" },
+        { "command": { "action": "newTab", "index": 8 }, "keys": "ctrl+shift+9" },
+        { "command": "nextTab", "keys": "ctrl+tab" },
+        { "command": "openNewTabDropdown", "keys": "ctrl+shift+space" },
+        { "command": "openSettings", "keys": "ctrl+," },
+        { "command": "paste", "keys": "ctrl+shift+v" },
+        { "command": "paste", "keys": "shift+insert" },
+        { "command": "prevTab", "keys": "ctrl+shift+tab" },
+        { "command": "resetFontSize", "keys": "ctrl+0" },
+        { "command": { "action": "resizePane", "direction": "down" }, "keys": "alt+shift+down" },

```

Comment:
i was hoping for consensus from @zadjii-msft before we made this change ;P but if he's fine with it and the reasoning, I am too

Reply:
Oh yea I'm cool with this, sorry for not replying earlier